### ISSUE TITLE
Version check error

### ISSUE THREAD

**Issue by DYPIXY** [Litium-org/litium#3]
**Error: src/native/engine/core/bios_versionCheck.lua:20: attempt to compare number with nil**
stack traceback:
        [love "boot.lua"]:345: in function '__lt'
        src/native/engine/core/bios_versionCheck.lua:20: in function 'doCheck'
        main.lua:41: in function 'load'
        [love "callbacks.lua"]:136: in function <[love "callbacks.lua"]:135>
        [C]: in function 'xpcall'
        [love "boot.lua"]:361: in function <[love "boot.lua"]:348>
        [C]: in function 'xpcall'

[ Writing about this error just to warn you, since you're working on version check ]
basically, the line `code, serverEngineVersion = request.newRequest("https://raw.githubusercontent.com/Litium-org/litium/master/.litversion")` sometimes returns nil or an incomplete value to the program progress in it execution, a good way to solve it is: 

try catch, error handling
or just comparing to nil (less elegant and can cause more errors but is pretty more simple)
or whatever you have on mind, having fun on this project :)

**Comment by itzKiwiSky** [comment]
fixed for 0.0.4, for now is comapring for nil
I trying reworking litium, more detail soon
